### PR TITLE
Add space.rirusha.Cassette

### PR DESCRIPTION
**Pull request by Rirusha**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.

Unofficial client of Yandex Music service on GTK4/Adwaita. 
Past version: https://github.com/flathub/io.github.Rirusha.Cassette, but I changed the id.

I know I need to add flathub.json to the above mentioned repository, like here https://github.com/flathub/org.gnome.iagno/blob/aefd92d1f0b783feedb62d101565e31f9c5dbc4b/flathub.json, then archive the repository.

Am I missing anything else?

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.

http://rirusha.space/

- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: space.rirusha.Cassette.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- /share/pkgconfig
+- /share/vala
+- '*.la'
+- '*.a'
+modules:
+- name: blueprint-compiler
+  buildsystem: meson
+  sources:
+  - type: git
+    url: https://gitlab.gnome.org/jwestman/blueprint-compiler
```
Comment (hfiguiere) [inline space.rirusha.Cassette.yml:38]: missing the cleanup rule to remove all of this build only dependency.

Comment (Rirusha) [reply space.rirusha.Cassette.yml:40]: My bad, missed that point in the documentation. Thanks for the correction

**Comment by bbhtt** [comment]
>  Past version: https://github.com/flathub/io.github.Rirusha.Cassette, but I changed the id.

That is already verified why do you need to change the ID ?

File: space.rirusha.Cassette.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- --share=network
+- --share=ipc
+- --socket=fallback-x11
+- --device=dri
+- --socket=pulseaudio
+- --socket=wayland
+- --filesystem=xdg-music:ro
+- --filesystem=xdg-download:ro
+- --talk-name=org.mpris.MediaPlayer2.Player
+- --own-name=org.mpris.MediaPlayer2.cassette
```
Comment (bbhtt) [inline space.rirusha.Cassette.yml:18]: Can you make this use `org.mpris.MediaPlayer2.space.rirusha.Cassette` ? Then this permission can be dropped.

Comment (Rirusha) [reply space.rirusha.Cassette.yml]: Done

**Comment by bbhtt** [comment]
> That is already verified why do you need to change the ID ?

This is not answered.

**Comment by Rirusha** [comment]
> > Past version: https://github.com/flathub/io.github.Rirusha.Cassette, but I changed the id.
> 
> That is already verified why do you need to change the ID ?

I moved the project to gitlab.gnome.org. I was supposed to change the id to io.gitlab.Rirusha.Cassette, but I decided to change to my domain so I wouldn't need to change it in the future.

**Pull request closed without merging**